chapters/sampling-randomization-power.tex

```diff
@@ -465,6 +459,13 @@ \subsection{Power calculations}
 simulation ensures you will have understood the key questions well enough
 to report standard measures of power once your design is decided.
 
+Not all studies are capable of achieving traditionally high power:
+sufficiently precise sampling or treatment assignments may not be available.
+This may be especially true for novel or small-scale studies --
+things that have never been tried before may be hard to fund or execute at scale.
+What is important is that every study includes reasonable estimates of its power,
+so that the evidentiary value of its results can be assessed.
+
```

Comment:
Did not expect to find a full paragraph edited this later into a chapter in this PR. Just want to make sure that this was intentional.

Reply:
I think this was caused by merging `develop` into here and then back. Not sure exactly how. If this isn't right we'll catch in review.

Reply:
I think it was intentional